base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
Sure, I think I've explained what I wanted to say clear enough so I can take any suggestions as long as the first sentense doesn't make it sound like this is doing a simple convertion and make it clear that the `x` is the output of `cconvert`.

By "the full document of the function" I mean I don't think it is better explained in the manual and I think it makes sense to describe it here in the doc string since it's a property of the function.